front/admin/cli.ts

```diff
@@ -462,17 +462,9 @@ const conversation = async (command: string, args: parseArgs.ParsedArgs) => {
       });
 
       if (response.isErr()) {
-        logger.error(response.error.message);
+        console.log(response.error.message);
```

Comment:
This code is a command to easily preview the rendering of a convo, I'm just switching to console log for more convenience. Command is 
```
./admin/cli.sh conversation render-for-model --wId=7ea8c3d99c --cId=b9e8lfTYKh
```